src/char_mapping.rs

```diff
@@ -6,6 +6,49 @@ use std::collections::HashMap;
 
 use {GlyphID, Result, Tape, Value};
 
+/// A mapping from a character code to a glyph id
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub enum Mapping {
+    U8(HashMap<u8, GlyphID>),
+    U16(HashMap<u16, GlyphID>),
+    U32(HashMap<u32, GlyphID>),
+    None,
+}
+
+impl Mapping {
+    /// Remove entries in the mapping which map to glyph id 0 (.notdef).
+    pub fn filter_empty_mappings(&mut self) {
+        match *self {
+            Mapping::U8(ref mut map) => map.retain(|_, v| v != &0),
+            Mapping::U16(ref mut map) => map.retain(|_, v| v != &0),
+            Mapping::U32(ref mut map) => map.retain(|_, v| v != &0),
+            Mapping::None => {}
+        }
+    }
+
+    /// Convenience function which converts the underlying mapping to a `HashMap<u32, GlyphID>`.
+    pub fn to_hashmap_u32(&self) -> HashMap<u32, GlyphID> {
```

Comment:
Again, it looks like something that makes testing easier. Move it to the tests.

Reply:
This function is actually not used by the tests. It was just for the convenience of the library user if he doesn't care about which character codes are actually available in this `Mapping`. Although thinking about it now, it probably doesn't fit this rather low-level library. I'm removing it.